fbw-a32nx/src/localization/flypad/en.json

```diff
@@ -489,10 +489,31 @@
       "PaxSigns": "PAX Signs",
       "RmpVhfSpacing": "RMP VHF Spacing",
       "Satcom": "Satcom",
+      "Select": "Select",
       "ThrustReductionHeight": "Thrust Reduction Height (ft)",
       "Title": "Aircraft Options / Pin Programs",
       "WeightUnit": "Weight Unit"
     },
+    "AutomaticCallOuts": {
```

Comment:
**thought:** I don't think we want the individual call-outs to be translated, as those are literally the vocalization you hear